### PR TITLE
Edit UI to fit layout

### PR DESCRIPTION
**Pull request by Reallyeasy1**
image

### CODE REVIEW

File: src/main/java/seedu/address/ui/StudentProfile.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    private Label studentIdLabel;
+    @FXML
+    private Label tutorialIdLabel;
+    @FXML
+    private FlowPane attendanceFlowPane;
+
+    /**
+     * Creates a {@code StudentProfile} to display the details of a student.
+     */
+    public StudentProfile() {
```
Comment (Fui03) [inline src/main/java/seedu/address/ui/StudentProfile.java:28]: I like how you implemented the UI to display student's details.

**Review by Fui03 — COMMENTED** [review]
LGTM! Try to pass the test case coverage!

File: src/main/resources/view/DarkTheme.css [hunk 2]
```diff
@@ -350,3 +350,72 @@
...
+}
+
+.attendance-date-label {
+    -fx-background-color: #4a4a4a; /* Dark grey for individual date labels */
+    -fx-text-fill: #cccccc; /* Light grey text */
+    -fx-padding: 3px 6px;
+    -fx-background-radius: 4px;
+    -fx-border-radius: 4px;
+    -fx-border-color: #5a5a5a; /* Medium grey border for a subtle highlight */
+    -fx-font-size: 12px;
```
Comment (tansanchian) [inline src/main/resources/view/DarkTheme.css:420]: Great that you give design for the attendance card!

**Review by tansanchian — APPROVED** [review]
LGTM!

**Pull request merged**